test/cpp/end2end/end2end_test.cc

```diff
@@ -64,6 +64,11 @@ using std::chrono::system_clock;
     }                   \
   } while (0)
 
+namespace grpc_impl {
+
+class ResourceQuota;
```

Comment:
This doesn't seem to be referenced later. Why is this one needed? For server_builder.cc for instance, we're clearly using the ResourceQuota later.

Reply:
https://github.com/grpc/grpc/blob/6dcf6d164510a66efa831f2620e6511ffda0eba1/test/cpp/end2end/end2end_test.cc#L1941

The class ResourceQuotaEnd2endTest has a member "ResourceQuota server_resource_quota_;" which requires this forward declaration. Since the whole other test is in namespace grpc, I moved the forward declaration out above this into namespace grpc_impl.